resources/sdk/clisdkclient/templates/root.mustache

```diff
@@ -35,9 +48,52 @@ var versionCmd = &cobra.Command{
 	Long:  `All software has versions. This is gc version's`,
 	Run: func(cmd *cobra.Command, args []string) {
 		fmt.Println("{{packageVersion}}")
+
+        var latestVersion string
+        if strings.Contains("{{basePath}}", "api.mypurecloud.com") {
+            latestVersion = retrieveLatestVersion(externalVersionURL)
+        } else {
+            latestVersion = retrieveLatestVersion(internalVersionURL)
+        }
+
+        if latestVersion == "{{packageVersion}}" {
```

Comment:
You'll have to write a function here to compare semantic version numbers. It would be a good idea to add a unit test to root_test to test the function.